test/Dialect/ESI/connectivity.mlir

```diff
@@ -82,3 +82,23 @@ hw.module @testIfaceWrap() {
   // CHECK-NEXT:     %6 = sv.modport.get %4 @Sink : !sv.interface<@IData> -> !sv.modport<@IData::@Source>
   // CHECK-NEXT:     esi.unwrap.iface %3 into %6 : (!esi.channel<i32>, !sv.modport<@IData::@Source>)
 }
+
+// -----
```

Comment:
Why did you split this test? I've only seen split input file used for error tests.

Reply:
I prefer tests to use `split-input-file`, erroring or not. To me, a split demarks a clear separation between what ops are involved in different tests and guides my focus. It also avoids unintentional namespace/aliasing issues.

I do see that i seem to be the only person checking in tests like this in the codebase (SCFToCalyx, Handshake, FSM, ...) - I'll revert the change in this file to avoid a style change in ESI.